orcid-core/src/main/java/org/orcid/core/salesforce/dao/impl/SalesForceDaoImpl.java

```diff
@@ -555,14 +555,8 @@ private String retrieveParentOrgNameFromSalesForce(String accessToken, String co
      * 
      */
     private List<Integration> retrieveIntegrationsFromSalesForce(String accessToken, String memberId) throws SalesForceUnauthorizedException {
-        String query = new String();
-        
-        if(Features.NEW_BADGES.isActive()) {
-            query = "SELECT Integration__c.Id, Integration__c.Name, Integration__c.Description__c, Integration__c.Integration_Stage__c, Integration__c.Level__c, Integration__c.BadgeAwarded__c, (Select Id, Name, Status__c, Badge__c from Achievements__r Where Status__c = 'Awarded') from Integration__c WHERE Integration__c.inactive__c=FALSE And Organization__c='%s'";
```

Comment:
@DanielPalafox please use this query on like 558, we want to keep this query and not the one in the `else` clause

Reply:
Ok